data/migrations/20200116153744_create-reactions-table.js

```diff
@@ -0,0 +1,28 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable("reactions", table => {
+      table.increments();
+      table.string("highlighted_text", 400).notNullable();
+      table.string("emoji", 400).notNullable();
```

Comment:
You should probably increase the character limit since someone could realistically highlight as many characters as possible.